intelmq/__init__.py

```diff
@@ -1,9 +1,11 @@
-SYSTEM_CONF_FILE = "/opt/intelmq/etc/system.conf"
-PIPELINE_CONF_FILE = "/opt/intelmq/etc/pipeline.conf"
-STARTUP_CONF_FILE = "/opt/intelmq/etc/startup.conf"
-RUNTIME_CONF_FILE = "/opt/intelmq/etc/runtime.conf"
+DEFAULT_LOGGING_LEVEL = "INFO"
+DEFAULT_LOGGING_PATH = "/opt/intelmq/var/log/"
 DEFAULTS_CONF_FILE = "/opt/intelmq/etc/defaults.conf"
 HARMONIZATION_CONF_FILE = "/opt/intelmq/etc/harmonization.conf"
-DEFAULT_LOGGING_PATH = "/opt/intelmq/var/log/"
-DEFAULT_LOGGING_LEVEL = "INFO"
+MODIFY_CONF_FILE = "/opt/intelmq/etc/modify.conf"
```

Comment:
Where should this file be located after installation? `/opt/intelmq/var/lib/bots/modify/modify.conf`?